src/Typeahead.svelte

```diff
@@ -18,6 +18,9 @@
 
   /** Set to `false` to prevent the first result from being selected */
   export let autoselect = true;
+  
+  /** Set to `keep` to keep the search field unchanged after select, set to `clear` to auto-clear search field */
```

Comment:
Is it "keep" or "clear"?

Reply:
actually I thought about 

- `keep` for not changing the search field
- `clear` for clearing the search field
- `update` for updating with the selection

But since there is no point in doing `value = value` the `keep` options doesn't show up later in the code

Reply:
Understood